examples/reliable_conversation/src/utils/log_formatter.py

```
@@ -0,0 +1,199 @@
+
+def format_message_content(content: str, max_line_length: int = 100) -> str:
+    """Format message content for better readability"""
+    if not content:
+        return content
+    
+    # Handle JSON strings in content
+    try:
+        if content.strip().startswith('{') and content.strip().endswith('}'):
+            parsed = json.loads(content)
+            return json.dumps(parsed, indent=2)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)